src/types/qwcompositor.cpp

```diff
@@ -8,6 +8,9 @@
 #include "qwoutput.h"
 #include "util/qwsignalconnector.h"
 #include "render/qwrenderer.h"
+#if WLR_VERSION_MINOR > 16
```

Comment:
only support 0.17/0.18 now

Reply:
ref:https://gitlab.freedesktop.org/wlroots/wlroots/-/releases 
key word: wp-fractional-scale-v1

Reply:
qwlroots do not support WLR_VERSION_MINOR <= 16,  just delete this line